client/platform/web-girder/views/Home.vue

```diff
@@ -167,12 +163,12 @@ export default {
           >
             <template #headerwidget>
               <run-pipeline-menu
-                :selected="(locationIsViameFolder ? [location] : selected)"
+                :selected="(locationIsViameFolder ? [location] : selectedViameFolders)"
```

Comment:
I believe this prop has been renamed `selected` to `selectedDatasetIds` or the preferred kebab case `selected-dataset-ids`